tests/system/feature_store/test_feature_store.py

```diff
@@ -141,6 +141,9 @@ def _ingest_quotes_featureset(self):
         self._logger.info(f"quotes spec: {quotes_set.spec.to_yaml()}")
         assert df["zz"].mean() == 9, "map didnt set the zz column properly"
         quotes_set["bid"].validator = MinMaxValidator(min=52, severity="info")
+        quotes_set["ticker"].validator = MinMaxLenValidator(
```

Comment:
i guess we should have a new test module with unit tests for all the validators with at least 1 good and 1 bad values (no need for full system test and we will get better covrage) 

i.e. wrap the validator with:

```python
feature = Feature(args...)
feature.validator = XxxValidator(args...)
feature.validator.set_feature(feature)
for value, expected in [(ok1, value1), ...]:
     ok, args = validator.check(value)
     assert ok = expected
```

Reply:
Let me do that

Reply:
@george0st i assume the `XxxValidator(args...)` can also go under the `for` loop to try different args (e.g. min/max/regex options)

Reply:
What about to provide first basic version with alignment and then extend ... (I see little different testing set for type check and other checks)